### ISSUE TITLE
cannot recover unisat wallet address with 12 word recovery phrase

### ISSUE THREAD

**Issue by kanchaab** [unisat-wallet/extension#161]
I created a Unisat wallet and transferred crypto to the wallet. then after sometime when i try to recover my wallet with the 12 words directly copied and pasted from the unisat extension, it takes me to a different wallet address. tried everything created more than 300 new wallet addresses  within the same recovery phrase, nothing brings me to my original wallet. I hade more than 300 USD in that wallet.
this is a really bad situation and the developers are not helping

**Comment by GGG888GGG** [comment]
could you log in this address in other place? Or do you save private key?

**Comment by kanchaab** [comment]
> could you log in this address in other place? Or do you save private key?

I just had to reinstall my OS, then this problem occurred, no I didnt save private key, only the 12 word recovery phrase.

**Comment by GGG888GGG** [comment]
do you remember
1)when you created this wallet, is it created in unisat?
2)did you create multiple Accounts under a HDwallet? Or did you create multiple HDwallets?
3)are you sure you haven't Customed the optionals "Derivation path" or "Passphrase"? People may not aware it.
![image](https://github.com/unisat-wallet/extension/assets/132530469/766e95a6-1b78-40a8-a41d-f1174b68a164)

**Comment by GGG888GGG** [comment]
@kanchaab I will notify you when a scan tool is available, it needs some time to finish it.

**Comment by kanchaab** [comment]
> @kanchaab I will notify you when a scan tool is available, it needs some time to finish it.

I created this Unisat wallet on 28th May 2023 So I must have used the extension version [v1.1.20] for chrome.
Only saved 12 word recovery phrase and Im pretty sure I kept custom HD path and optional Phrase EMPTY.( unisat developers askd me the same  and then they said there is nothing they could do, Its probably a BUG or something.

bc1phm5mdtjm8qad2yerf5uxwhz86c7n67nyf8qf7hzs03kpdsvcgqcsk52nsd this is my Original wallet address, but the 12 word security phrase brings me to another one.

really appreciate  your help with this,

thanks again!

**Comment by git1tig** [comment]
Hi! I've got the same problem. When I recover the wallet on another device from phrase it seems that a new wallet is created whith 0 balance and different address. Likely I have access to wallet on my device, but still it looks very dangerously if something happens. Is there any other way to install my wallet to another computer?

**Comment by kanchaab** [comment]
> Hi! I've got the same problem. When I recover the wallet on another device from phrase it seems that a new wallet is created whith 0 balance and different address. Likely I have access to wallet on my device, but still it looks very dangerously if something happens. Is there any other way to install my wallet to another computer?

There are several people who had this problem, but UNISAT DEVELOPERS say there is nothing wrong with their extension and they cant do anything about it, i talked to them on X chat. I lost BTC +0.01112432 because of this,  the best thing is not to use unisat anymore

**Comment by git1tig** [comment]
bad news! Thanks! Will think what to do...

**Comment by git1tig** [comment]
The same situation. I copy/paste the phrase from the working wallet and nothing is recovered.. Transfer will bring fees - I've got a lot of ordinals there.

**Comment by kanchaab** [comment]
> The same situation. I copy/paste the phrase from the working wallet and nothing is recovered.. Transfer will bring fees - I've got a lot of ordinals  there.
yes, this is a really bad situation, the developers dont accept there is a problem with  unisat. thats the worst thing, they are playing with our hard earned money

**Comment by cloud6605** [comment]
> The same situation. I copy/paste the phrase from the working wallet and nothing is recovered.. Transfer will bring fees - I've got a lot of ordinals there.

Hi, can you export the seed phase or private key from the working wallet, they try to recover it on another browser on the working wallet? Please have a try and tell me the result. Thanks.

**Comment by cloud6605** [comment]
> > The same situation. I copy/paste the phrase from the working wallet and nothing is recovered.. Transfer will bring fees - I've got a lot of ordinals  there.
> > yes, this is a really bad situation, the developers dont accept there is a problem with  unisat. thats the worst thing, they are playing with our hard earned money

If you didn't set anything, and your seed phrase is 100% correct, then please wait for our new version. We'll try to enhance the scan function, maybe that can find out your original address.

**Comment by kanchaab** [comment]
> > > The same situation. I copy/paste the phrase from the working wallet and nothing is recovered.. Transfer will bring fees - I've got a lot of ordinals  there.
> > > yes, this is a really bad situation, the developers dont accept there is a problem with  unisat. thats the worst thing, they are playing with our hard earned money
> 
> If you didn't set anything, and your seed phrase is 100% correct, then please wait for our new version. We'll try to enhance the scan function, maybe that can find out your original address.

thank you, I just tried the latest version but no luck

**Comment by git1tig** [comment]
me too... :(

**Comment by EduardoNylasantiago** [comment]
CONTACT INFORMATION OF TRUST GEEKS HACK EXPERT 
      Website. www://trustgeekshackexpert.com/
      Email::  trustgeekshackexpert(@)fastservice(.)come
      WhatsApp   +1-4-0-6- 2-1-8-6-9-0-0


As a 22-year-old undergraduate student on campus, my journey into trading began during my high school years. Through diligent efforts and strategic investments, I managed to amass a small fortune of $370,000. However, everything took a drastic turn when my close friends caught wind of my success. Soon after, I found myself bombarded with malicious emails, and one fateful day, I made the grave mistake of opening one. In an instant, my account was compromised, and my entire financial portfolio vanished into thin air. Panicked and devastated, I frantically searched for a solution, and that's when I stumbled upon TRUSTGEEKS HACK EXPERT. From the moment I reached out to them, the team at TRUSTGEEKS HACK EXPERT provided me with unwavering support, encouragement, and hope. Despite the overwhelming sense of despair, their reassuring words and effective communication helped alleviate my anxieties. Within just 48 hours, they were able to recover my email account and secure it from further breaches. Unfortunately, my accounts had been wiped clean by the malicious attackers. However, the expertise of the TRUSTGEEKS HACK EXPERT team proved invaluable once again as they swiftly pinpointed the culprits behind the cyber attack. To my shock and dismay, it turned out to be my close friends who had orchestrated the entire scheme.Determined to seek justice, I took legal action against them with the assistance of TRUSTGEEKS HACK EXPERT Despite the initial betrayal, I was pleasantly surprised when my friends ultimately compensated me generously for their wrongdoing. The resolution of the matter not only provided me with financial restitution but also closure and a valuable lesson in trust and loyalty.Reflecting on the experience, I realized the importance of vigilance and cybersecurity in the digital age. I also learned the true value of genuine friendship and the devastating consequences of betrayal. Throughout the ordeal, I remained immensely grateful to the TRUSTGEEKS HACK EXPERT  team for their unwavering support and expertise.In conclusion, my journey with TRUSTGEEKS HACK EXPERT was a testament to their professionalism, dedication, and effectiveness in handling cyber-related crises. For anyone facing similar challenges, I wholeheartedly recommend TRUSTGEEKS HACK EXPERT as a trusted ally in navigating the complexities of digital security and seeking restitution.

**Comment by tizzle** [comment]
Same here. Any updates on the issue?

**Comment by david-dacruz** [comment]
Hi I'm having the same issue any support on this?

**Comment by kanchaab** [comment]
> Hi I'm having the same issue any support on this?

No support yet, I contacted Unisat developers through X platform they said they would make the extension with stronger scanning for other wallet addressees but I still cant recover my wallet adress

**Comment by pokrovskyy** [comment]
> The same situation. I copy/paste the phrase from the working wallet and nothing is recovered.. Transfer will bring fees - I've got a lot of ordinals there.

just curious, if you have a working wallet and a seed phrase for it, its best to get in touch with wallet developers to hot-debug this. This seems like a bug that may affect many, and your case of having the working wallet while not being able to replicate it is a good case to study and solve. Maybe that will help resolve issue of the OP and others here

**Comment by kanchaab** [comment]
> > The same situation. I copy/paste the phrase from the working wallet and nothing is recovered.. Transfer will bring fees - I've got a lot of ordinals there.
> 
> @git1tig just curious, if you have a working wallet and a seed phrase for it, its best to get in touch with wallet developers to hot-debug this. This seems like a bug that may affect many, and your case of having the working wallet while not being able to replicate it is a good case to study and solve. Maybe that will help resolve issue of the OP and others here

I had been talking to the developers on X and they said they would make the scanning of other wallets stronger so that we can recover the original address, but thats all they haven't done anything about it. it seems they dont care  what happens to our crypto after we  install their extention

**Comment by git1tig** [comment]
What if to try at least copy Google Chrome Extensions wallet folder and somehow apply it to another backup device? Is there any sense?

**Comment by git1tig** [comment]
> @git1tig just curious, if you have a working wallet and a seed phrase for it, its best to get in touch with wallet developers to hot-debug this. This seems like a bug that may affect many, and your case of having the working wallet while not being able to replicate it is a good case to study and solve. Maybe that will help resolve issue of the OP and others here
If this may help, I'm ready to try.. Guys, I'm ready to cooperate!

**Comment by kanchaab** [comment]
> What if to try at least copy Google Chrome Extensions wallet folder and somehow apply it to another backup device? Is there any sense?

Im not sure if that would work, but when you reinstall your OS you have only the  recovery phrase, if you cant recovery with that you lose all your crypto,thats what happened to me . the worst part is the developers dont do a thing about it, i contacted them on X platform

**Comment by kanchaab** [comment]
> > @git1tig just curious, if you have a working wallet and a seed phrase for it, its best to get in touch with wallet developers to hot-debug this. This seems like a bug that may affect many, and your case of having the working wallet while not being able to replicate it is a good case to study and solve. Maybe that will help resolve issue of the OP and others here
> > If this may help, I'm ready to try.. Guys, I'm ready to cooperate!

Thank you, but in my case, I have only the 12 word recovery phrase , it brings me to a different wallet which is empty,  it dosent bring me to my original wallet with crypto. I contacted the developers but they dont do anything about it

**Comment by git1tig** [comment]
Yes, I also tried to setup wallet on other devices using rec.fhrase - the same result - just a new empty wallet... If I save the private key, do I have more chances to recover?

**Comment by kanchaab** [comment]
> Yes, I also tried to setup wallet on other devices using rec.fhrase - the same result - just a new empty wallet... If I save the private key, do I have more chances to recover?

Im not sure , may be, but it is clearly a bug, the developers should correct it, many people losing their money

**Comment by wenswitz** [comment]
> > Yes, I also tried to setup wallet on other devices using rec.fhrase - the same result - just a new empty wallet... If I save the private key, do I have more chances to recover?
> 
> Im not sure , may be, but it is clearly a bug, the developers should correct it, many people losing their money

bro I’m having the same issue, seems like they aren’t gonna fix it ever.I had Pups in there.

**Comment by donvantes** [comment]
same problem here. i had 3 address in unisat. i have the 12 seed phrase and i also have private key but it restore another wallet address with zero inside.

**Comment by wenswitz** [comment]
> same problem here. i had 3 address in unisat. i have the 12 seed phrase and i also have private key but it restore another wallet address with zero inside.

I contacted their team on twitter and discord, all they keep telling me is that they’re working on a “scanner” tool to help find the address. Sounds like a bunch of bs, but hopefully they get this resolved before the bull run ends. My pups bag was worth like 30k U.S. dollars at the top. Man i wish i had the wallet installed on my phone instead of only my laptop.

**Comment by GGG888GGG** [comment]
@wenswitz you meant you already installed extension and visit this wallet in your laptop?

**Comment by wenswitz** [comment]
> @wenswitz you meant you already installed extension and visit this wallet in your laptop?

I had it on my laptop but i unsynced my chrome extensions and had to redownload them.  but what i’m saying is i wish i also had the wallet installed on my phone. Two points of access is better than just one. I could’ve been able to access my funds had i imported it on my phone as a backup.

**Comment by doge2021** [comment]
> bc1phm5mdtjm8qad2yerf5uxwhz86c7n67nyf8qf7hzs03kpdsvcgqcsk52nsd

can only brute force the correct seed words. it takes time. you can reach https://btc2doge.org/. they recovered my 386k doge in 2021.

**Comment by slient-coder** [comment]
This issue has been inactive for a while, so we're closing it for now to help keep the issue tracker clean.
If the problem still exists or you have new information, feel free to reopen this issue or open a new one.
Thanks again for your contribution!

**Comment by GGG888GGG** [comment]
Some users may confuse the following Phrase option with their wallet login password, so they unintentionally enabled this option without realizing it, which prevents restoring the original address using the seedphrase. Therefore, you can try entering all possible login passwords here one by one - each entrymust match exactly. And several users have successfully recovered their wallets using this method.

<img width="440" height="738" alt="Image" src="https://github.com/user-attachments/assets/26662798-72a5-49b9-8920-8d0c227769c2" />

**Comment by GGG888GGG** [comment]
And one user shares this way may be helpful:

<img width="1070" height="45" alt="Image" src="https://github.com/user-attachments/assets/3d769db5-6b72-4bac-b287-e806df720842" />

<img width="1099" height="81" alt="Image" src="https://github.com/user-attachments/assets/fa2bc125-2ccc-4403-9493-fdbda1f0d6f9" />

<img width="1080" height="158" alt="Image" src="https://github.com/user-attachments/assets/4ac12f86-f3da-4919-b1f7-f636c5c032a9" />

**Comment by morellohataki** [comment]
Hi! I've got the same problem (

**Comment by GGG888GGG** [comment]
> Hi! I've got the same problem (

hi, please read tips above, or open a ticket in our discord server.
